.github/workflows/ios-android-test.yml

```diff
@@ -37,6 +37,9 @@ jobs:
               working-directory: platform/ios
               run: make test
             - uses: actions/setup-java@v2
+              if: |
+                !github.head_ref
+                && ( github.ref == 'refs/heads/release' || github.ref == 'refs/heads/main' )
```

Comment:
It's supposed to be `startsWith('release')` and `== 'main'` - since we have `release-*` branches too. Also needs fixing here: https://github.com/polypoly-eu/polyPod/pull/786/commits/990eb4644ad3351615518dabb2a25fdd408a24e6